_posts/2024-05-18-developing-openwrt-in-qemu.adoc

```diff
@@ -0,0 +1,88 @@
+---
```

Comment:
@olexii4 Let's change the publication date to 05.22

Reply:
I have fixed it